bundles/org.openhab.ui/web/src/assets/definitions/widgets/system/knob.js

```diff
@@ -11,5 +11,7 @@ export default () => [
   pt('secondaryColor', 'Secondary Color', 'Color of the rest of the control (HTML value, default #dcdfe6)'),
   pt('textColor', 'Text Color', 'Color of the value text (HTML value, default #000000)'),
   pt('strokeWidth', 'Stroke Width', 'Thickness of the arcs, default 17'),
-  pb('responsive', 'Responsive', 'Size the control using percentages instead of pixels')
+  pb('responsive', 'Responsive', 'Size the control using percentages instead of pixels'),
+  pn('updateInterval', 'Update Interval', 'Time to wait between subsequent commands in ms (default 500)').a(),
```

Comment:
Could you also regenerate the docs for the added properties? - a number of components based on the knob and slider are affected.
(I really need to work to have this done automatically).

Reply:
done.